src/process/process/libs/threadpool/impl/default/threadpool_impl.cpp

```diff
@@ -2,136 +2,138 @@
 
 #include "logging.hpp"
 
-namespace threadpool {
-Threadpool_impl::Threadpool_impl()
-    : Threadpool_impl(std::thread::hardware_concurrency()) {
-  Log(LogLevel::debug) << LOG_START;
-  Log(LogLevel::debug) << LOG_END;
-}
+namespace process {
```

Comment:
Same as for scheduler, threadpool is a library and should not be in the process namespace